lib/tasks/one-off/doc_sentencing_counties.rake

```diff
@@ -0,0 +1,23 @@
+require 'open-uri'
+
+namespace :doc do
+  desc 'Populates DocSentencingCounty and assignes it to Doc::Sentence'
+  task :sentencing_counties do
+    sent_counties.each do |sent|
```

Comment:
Having an issue with this line. Where does the sent_counties come from?

Reply:
I believe sent_counties was accidentally deleted. added it back in